src/sigchain/Sigchain.ts

```diff
@@ -220,7 +220,9 @@ class Sigchain {
    * Appends a claim (of any type) to the sigchain.
    */
   @ready(new sigchainErrors.ErrorSigchainNotRunning())
-  public async addClaim(claimData: ClaimData): Promise<ClaimEncoded> {
+  public async addClaim(
```

Comment:
Were these brought in from @emmacasolin in #311

Reply:
I made `addClaim()` return a `ClaimEncoded` in https://github.com/MatrixAI/js-polykey/pull/278 in order to implement the `identities claim` command, and then `ClaimId` was added here https://github.com/MatrixAI/js-polykey/pull/292/commits/3dea67340cb83a4e385a934a6f4323a67f19096c. But we were talking on Slack about how probably only `ClaimId` needs to be returned.

Reply:
I pulled it in from qa-testing to fix a problem when I matched dependency versions with qa-testing.